modules/installation-creating-gcp-firewall-rules-vpc.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-firewall-rules-vpc_{context}"]
+= Creating firewall rules in GCP
+
+You must create firewall rules Google Cloud Platform (GCP) for your
+{product-title} cluster to use. One way to create these components is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your GCP
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and associated subnets in GCP.
+
+.Procedure
+
+. Copy the template from the
+*Deployment Manager template for firewall rules*
+section of this topic and save it as `03_firewall.py` on your computer. This
+template describes the security groups and roles that your cluster requires.
+
+. Export the following variables required by the resource definition:
++
+----
+$ export MASTER_NAT_IP=`gcloud compute addresses describe ${INFRA_ID}-master-nat-ip --region ${REGION} --format json | jq -r .address`
+$ export WORKER_NAT_IP=`gcloud compute addresses describe ${INFRA_ID}-worker-nat-ip --region ${REGION} --format json | jq -r .address`
+----
+
+. Create a `03_firewall.yaml` resource definition file:
++
+----
+$ cat <<EOF >03_firewall.yaml
+imports:
+- path: 03_firewall.py
+
+resources:
+- name: cluster-firewall
+  type: 03_firewall.py
+  properties:
+    allowed_external_cidr: '0.0.0.0/0' <1>
```

Comment:
The value of allowed_external_cidr differs between external cluster and internal cluster. For external cluster, it's '0.0.0.0/0'. For internal cluster, it's '${NETWORK_CIDR}'